problems/numberOfDigits.js

```diff
@@ -7,8 +7,9 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    let i = n.toString()
+    return i.length
```

Comment:
How can you condense this code to look cleaner? Consider chaining and implicit returns.